apps/editing-toolkit/editing-toolkit-plugin/wpcom-tasks/index.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Atomic support for WPCOM onboarding checklist tasks.
+ *
+ * @package A8C\FSE
+ */
+
+namespace A8C\FSE\WPCOMTasks;
+
+/**
+ * This logic is duplicated from Onboarding_Checklist\Blog
+ */
+function page_post_saved( $post_ID, $post, $update ) {
+	if ( get_option( 'show_on_front' ) === 'page' && (int) get_option( 'page_on_front' ) === $post_ID ) {
+		update_option( 'onboarding_checklist_task_front_page_updated', true );
+	}
+}
+add_action( 'save_post_page', __NAMESPACE__ . '\\page_post_saved', 10, 3 );
+
+/**
+ * This logic is duplicated from Onboarding_Checklist\Blog
+ */
+function update_option_page_on_front( $old_value, $new_value ) {
+	if ( $old_value === $new_value ) {
+		return;
+	}
+
```

Comment:
I'm not sure if removing the `_hs_front_page` page meta is necessary for atomic sites? It seems to have worked so far without it.